jstz_cli/src/accounts/account_list.rs

```diff
@@ -0,0 +1,39 @@
+use std::collections::HashMap;
```

Comment:
I don't really see a need for `AccountList`, please inline the `HashMap<String, Account>` type in `Config`

Reply:
The idea is that it should be an abstraction for account list hiding its actual implementation - maybe a better name would be AccountConfig. Right now it just wraps the hashmap, but later I wanted to extend it with modification of self-address, methods for printing all accounts, some methods of type "find me an account in the list that has this alias" etc.

Reply:
> but later I wanted to extend it with modification of self-address
You mean storing the 'current' account, which is set with `jstz login` and printed with `jstz whoami`?
> methods for printing all accounts
Likely not that best place to put that, observable behaviour (such as printing) should be attached with command code
> some methods of type "find me an account in the list that has this alias" etc.
Makes sense

Given this, I'd suggest moving `AccountList` to `config.rs` with the type `AccountConfig`

Reply:
instead of printing I think there could be a method like to_string which would then be used by the command code and printed. I can move it to the config - it seems best to me to create a folder config with config.rs and accountconfing.rs and sandboxconfig.rs.

Reply:
I think given we have `SandboxConfig` and `Config` in `config`, we should probably add `AccountConfig` to here. 

Additionally while these structs have enough functionality to warrant a type, I don't think they'll have enough functionality to warrant a module.